lib/tasks/docker.rake

```diff
@@ -14,6 +14,8 @@
 # Other useful environment variables (not specific to this rake task)
 # => COMMIT_HASH    used by the discourse_test docker image to load a specific commit of discourse
 #                   this can also be set to a branch, e.g. "origin/tests-passed"
+# => SKIP_LINT      if using the discourse_test docker image, this will skip the docker:test task, and only run the docker:lint task
```

Comment:
Sure, when I added this documentation I was trying to have a "single place" where all the info for using the docker container could be found. Note that this is under the "not specific to this rake task" heading. That said, it would be useful to have it in docker_test.rb as well - I will copy the relevant stuff over later today.